Add account

shreyasananth-3 opened the pull request.

shreyasananth-3 committed:
feat: add-new-account-pages-and-assets-modify-existing-components

shreyasananth-3 committed:
feat: add account all pages

shreyasananth-3 committed:
feat: add account all pages

shreyasananth-3 committed:
feat: add account all pages

shreyasananth-3 committed:
feat: add account all pages

shreyasananth-3 committed:
feat: change to be made

advp7 committed:
feat: develop add account sync

shreyasananth-3 committed:
feat: feedback correction updated

advp7 committed:
feat: update add-account

advp7 committed:
Merge branch 'develop' of https://github.com/ShreyasGeetha/cypherock-cysync into feat/dialog-account

advp7 committed:
feat: add connect device dialog

advp7 committed:
Merge branch 'develop' of https://github.com/ShreyasGeetha/cypherock-cysync into feat/dialog-account

shreyasananth-3 committed:
feat: dialog transition

shreyasananth-3 committed:
feat: dialog transition

shreyasananth-3 committed:
feat: dialog transition

shreyasananth-3 committed:
feat: dialog transition

advp7 committed:
feat: add keypress to transition

shreyasananth-3 committed:
feat: dialog transition

shreyasananth-3 committed:
feat: dialog transition

shreyasananth-3 committed:
feat: dialog transition

shreyasananth-3 committed:
Merge pull request #1 from ShreyasGeetha/feat/addAccDiagMgmt

Feat/add acc diag mgmt

shreyasananth-3 committed:
feat: dialog transition

shreyasananth-3 committed:
feat: dialog transition

shreyasananth-3 committed:
feat: dialog transition

shreyasananth-3 committed:
feat: dialog transition

shreyasananth-3 committed:
feat: checkbox issue

shreyasananth-3 committed:
Update LeanBox.tsx

remove multiple onclick and embedded everything under Label tag

shreyasananth-3 committed:
feat: updating with respect to coding standards

shreyasananth-3 committed:
feat: updating with respect to coding standards

shreyasananth-3 committed:
feat: updating with respect to coding standards

shreyasananth-3 committed:
Update packages/cysync-core/src/dialogs/AddAccountGuide/Dialogs/Device/AddAccountSingleChainDialog.tsx

Co-authored-by: irshadCypherock <80461749+irshadCypherock@users.noreply.github.com>

shreyasananth-3 committed:
Update packages/cysync-core/src/constants/i18n/en.ts

Co-authored-by: irshadCypherock <80461749+irshadCypherock@users.noreply.github.com>

shreyasananth-3 committed:
feat: pr review altered code

shreyasananth-3 committed:
feat: pr review altered code

shreyasananth-3 committed:
feat: merge develop into addAccount

shreyasananth-3 committed:
feat: merge develop into addAccount

shreyasananth-3 committed:
feat: removing errors from console

shreyasananth-3 committed:
feat: dropdown hover

shreyasananth-3 committed:
feat: dropdown hover

shreyasananth-3 committed:
feat: dropdown hover

shreyasananth-3 committed:
feat: final PR changes

shreyasananth-3 committed:
feat: removing a comment

shreyasananth-3 committed:
feat: final fixes

shreyasananth-3 committed:
Merge branch 'develop' into addAccount

shreyasananth-3 committed:
feat: button disabled fix and dropdown background color on select

shreyasananth-3 committed:
feat: dropdown color

irshadCypherock committed:
Merge branch 'develop' into addAccount

shreyasananth-3 committed:
feat: adding a changeset

shreyasananth-3 committed:
feat: data array of InitialiseAccount changed

shreyasananth-3 committed:
feat: changes from UI team feedback

irshadCypherock commented on packages/cysync-core/src/dialogs/AddAccountGuide/Dialogs/index.ts:
```diff
@@ -1,3 +1,3 @@
 export * from './Asset';
-export * from './Device';
+export * from './X1 Vault';
```
Keep this `Device`, `X1 Vault` is only for display texts, not for folder and file names.

shreyasananth-3 replied:
Done

shreyasananth-3 committed:
feat: scrollable dialogbox

shreyasananth-3 committed:
feat: design changes and arrow movements with tab key

shreyasananth-3 commented on packages/ui/src/components/atoms/Checkbox.tsx:
```diff
@@ -61,6 +61,11 @@ const CheckBoxLabelStyle = styled.label.attrs<ISize>(props => ({
+
+  ${CheckBoxStyle}:focus + & {
+    outline: 2px solid ${({ theme }) => theme.palette.background.sideBar};
+    outline: 2px solid #eef209;
```
As mentioned line 67 works but 66, even the style doesnt get applied. Need your help on whether am I making any mistake with respect to template literal @irshadCypherock

irshadCypherock replied:
It won't work since `sideBar` is a gradient.

shreyasananth-3 replied:
background.sidebar is hex value, its there in color.styled
    sidebar: '#1E1A15',

shreyasananth-3 replied:
Changed the logic and handled it differently

shreyasananth-3 committed:
Update dropdown.util.tsx

updated dropdown util so that event propagation is handled except for escape key

shreyasananth-3 committed:
Merge branch 'develop' into addAccount

irshadCypherock commented on packages/ui/src/components/utils/index.ts:
```diff
@@ -55,4 +55,4 @@ export * from './height.styled';
 export * from './border.styled';
 export * from './Gradient';
-export * from './MenuFilter';
+export * from './dropdown.util';
```
```suggestion
export * from './dropdown';
```

shreyasananth-3 replied:
done

irshadCypherock commented on packages/ui/src/components/atoms/index.ts:
```diff
@@ -17,6 +17,6 @@ export * from './Select';
 export * from './Tag';
 export * from './Toggle';
-export * from './Dropdown';
+export * from './Dropdown/Dropdown';
```
```suggestion
export * from './Dropdown';
```

shreyasananth-3 replied:
done

irshadCypherock commented on packages/ui/src/components/atoms/Dropdown/Dropdown.tsx:
```diff
@@ -0,0 +1,214 @@
+  disabled = false,
+}) => {
+  const [search, setSearch] = useState('');
+  const [isOpen, setisOpen] = useState(false);
```
```suggestion
  const [isOpen, setIsOpen] = useState(false);
```

shreyasananth-3 replied:
done

shreyasananth-3 committed:
feat: removed DialogHeaders

shreyasananth-3 committed:
feat: optimizing dropdown menu

shreyasananth-3 committed:
feat: optimizing dropdown menu

shreyasananth-3 committed:
feat: optimizing dropdown menu

shreyasananth-3 committed:
feat: added throbber for loading

shreyasananth-3 committed:
feat: dropdown disabled icon color change

shreyasananth-3 committed:
feat: dropdown disabled icon color change

shreyasananth-3 committed:
feat: resolving merge conflicts with develop

shreyasananth-3 commented:
Even after resolving conflict locally, its still not updating

shreyasananth-3 commented:
trying to see whether reopening its reflecting new changes
